x/escrow/codec.pb.go

```diff
@@ -33,49 +46,17 @@ type Escrow struct {
 	Sender    []byte `protobuf:"bytes,1,opt,name=sender,proto3" json:"sender,omitempty"`
 	Arbiter   []byte `protobuf:"bytes,2,opt,name=arbiter,proto3" json:"arbiter,omitempty"`
 	Recipient []byte `protobuf:"bytes,3,opt,name=recipient,proto3" json:"recipient,omitempty"`
-	// amount may contain multiple token types
-	Amount []*coin.Coin `protobuf:"bytes,4,rep,name=amount" json:"amount,omitempty"`
 	// if unreleased before timeout, will return to sender
 	// timeout stored here is absolute block height
 	Timeout int64 `protobuf:"varint,5,opt,name=timeout,proto3" json:"timeout,omitempty"`
 	// max length 128 character
-	Memo                 string   `protobuf:"bytes,6,opt,name=memo,proto3" json:"memo,omitempty"`
-	XXX_NoUnkeyedLiteral struct{} `json:"-"`
-	XXX_sizecache        int32    `json:"-"`
-}
-
-func (m *Escrow) Reset()         { *m = Escrow{} }
```

Comment:
You are using a different version of protoc to compile.
Please use the vendored version.

It would be good to run `make prototools` again, then `make protoc`.
Also, probably `rm ${GOPATH}/bin/protoc_gen_gogo*` before `make prototools` to make sure it recompiles correctly
These methods should be available when using the vendored configs.

Reply:
🙈 will do, thanks!